packages/foundation/src/atoms/roles/colors.css

```diff
@@ -0,0 +1,8 @@
+html * {
+    /* LIGHT */
+    /* STATIC */
+    --o-ui-static-white-light: var(--o-ui-white);
+    /* DARK */
+    /* STATIC */
+    --o-ui-static-white-dark: var(--o-ui-white);
```

Comment:
Not sure I get this one.

You told me static is used to identify a color that doesn't change when switching between theme.

So why having a light & dark static white?

I would have expected a single prop called "o-ui-static-white"

Reply:
it was for consistency, but this is a valid point